demos/object_detection_demo_ssd_async/main.cpp

```diff
@@ -1,41 +1,126 @@
+static const char model_message[] = "Required. Path to an .xml file with a trained model.";
+static const char target_device_message[] = "Optional. Specify the target device to infer on (the list of available devices is shown below). "
+"Default value is CPU. Use \"-d HETERO:<comma-separated_devices_list>\" format to specify HETERO plugin. "
+"The demo will look for a suitable plugin for a specified device.";
+static const char labels_message[] = "Optional. Path to a file with labels mapping.";
+static const char performance_counter_message[] = "Optional. Enables per-layer performance report.";
+static const char custom_cldnn_message[] = "Required for GPU custom kernels. "
+"Absolute path to the .xml file with the kernel descriptions.";
+static const char custom_cpu_library_message[] = "Required for CPU custom layers. "
+"Absolute path to a shared library with the kernel implementations.";
+static const char thresh_output_message[] = "Optional. Probability threshold for detections.";
+static const char raw_output_message[] = "Optional. Inference results as raw values.";
+static const char input_resizable_message[] = "Optional. Enables resizable input with support of ROI crop & auto resize.";
+static const char num_inf_req_message[] = "Optional. Number of infer requests.";
+static const char num_threads_message[] = "Optional. Number of threads.";
+static const char num_streams_message[] = "Optional. Number of streams to use for inference on the CPU or/and GPU in "
+"throughput mode (for HETERO and MULTI device cases use format "
+"<device1>:<nstreams1>,<device2>:<nstreams2> or just <nstreams>)";
+static const char no_show_processed_video[] = "Optional. Do not show processed video.";
+static const char utilization_monitors_message[] = "Optional. List of monitors to show initially.";
+static const char iou_thresh_output_message[] = "Optional. Filtering intersection over union threshold for overlapping boxes (YOLOv3 only).";
+static const char mt_message[] = "Model type: ssd or yolo";
+
+DEFINE_bool(h, false, help_message);
+DEFINE_string(i, "", video_message);
+DEFINE_string(m, "", model_message);
+DEFINE_string(d, "CPU", target_device_message);
+DEFINE_string(labels, "", labels_message);
+DEFINE_bool(pc, false, performance_counter_message);
+DEFINE_string(c, "", custom_cldnn_message);
+DEFINE_string(l, "", custom_cpu_library_message);
+DEFINE_bool(r, false, raw_output_message);
+DEFINE_double(t, 0.5, thresh_output_message);
+DEFINE_double(iou_t, 0.4, iou_thresh_output_message);
+DEFINE_bool(auto_resize, false, input_resizable_message);
+DEFINE_uint32(nireq, 2, num_inf_req_message);
+DEFINE_uint32(nthreads, 0, num_threads_message);
+DEFINE_string(nstreams, "", num_streams_message);
+DEFINE_bool(loop, false, loop_message);
+DEFINE_bool(no_show, false, no_show_processed_video);
+DEFINE_string(u, "", utilization_monitors_message);
+DEFINE_string(mt, "", mt_message);
```

Comment:
Either pick a default value or make the error message clearer. Now it looks weird: `Invalid model type provided:`.

Reply:
OK, changed